util.js

```diff
@@ -640,4 +640,20 @@
         return url && url.replace(/^https/, "http");
     };
 
+    /**
+     * Unescapes HTML entities so the text in which they appear can be displayed as text of DOM elements
+     * 
+     * This implementation is as suggested by the MDN:
+     * https://developer.mozilla.org/en-US/Add-ons/Code_snippets/HTML_to_DOM
+     *
+     * @param {String} html foto escape entities
+     * 
+     * @return {String}
+     */
+    DDG.unescape = function(text) {
+        var doc = document.implementation.createHTMLDocument("example");
```

Comment:
It's roughly 1.7x slower than the method in the subreddit IA, which is not perfect but isn't prohibitively slow. Plus, this  bit from the MDN docs pretty much convinced me:

> The function below will safely parse simple HTML and return a DOM object which can be manipulated like web page elements. This will remove tags like `<script>`, `<style>`, `<head>`, `<body>`, `<title>`, and `<iframe>`. It will also remove all JavaScript, including element attributes that contain JavaScript.

That's a pretty good find, @jagtalon and @moollaza! Thanks for looking into it.